plugins/jira-dashboard/README.md

```diff
@@ -61,6 +61,23 @@ const defaultEntityPage = (
 );
 ```
 
+3. OPTIONAL - List of user issues on the homepage can be enabled by this code in your `app/src/components/home/Homepage.tsx` :
```

Comment:
IMHO, this should have it's own header? So it's easier to differentiate from the Dashboard components. Or what do you think?

Reply:
I added ### level, hopefully it's more clear now.